### ISSUE TITLE
Unable to interact with blocks that require an empty hand while holding something in the offhand.

### ISSUE THREAD

**Issue by Fokson** [Nuchaz/BiblioCraft#428]
Just a minor gripe, but it's annoying to have to remove my shield to open a display case (or do anything that requires an empty hand).  Could you instead check only the main hand when interacting/shift-interacting with a bibliocraft block?

**Comment by CovertJaguar** [comment]
This also affects armor stands.

**Comment by Nuchaz** [comment]
This issue seems to go away in minecraft v1.11 without any changes. I checked and I am only checking the main hand and on the display case I don't even check what is in the player hand to open/close the case, I only check that its on the server and the player is sneaking. It seems like if the player is sneaking, the method on the ItemBlock / Item is in your offhand is called before anything with your main hand. I'm not sure what I can do to fix this in the version for MC 1.10.2, but it seems to be fixed in the version for MC 1.11.2 that I got coming soon.
